### ISSUE TITLE
Settings Page is Loading Forever

### ISSUE THREAD

**Issue by hassanselim0** [docker/for-win#6211]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 740ABFC4-4FD7-42DF-A3EA-FDA214A8EE32/20200402185943

### Expected behavior
An accessible settings page inside Docker Desktop Dashboard

### Actual behavior
Settings page shows a loading spinner overlay that stays there forever, I can't click on anything other than "Cancel" and the "x" button

### Information

  - Windows Version: 10 version 1909 (Build 18363)
  - Docker Desktop Version: 2.2.0.5 (also happened on 2.2.0.4, didn't test on earlier 2.2.0.x)
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No

### Steps to reproduce the behavior

1. Open Docker Desktop UI
2. Click on Settings
![image](https://user-images.githubusercontent.com/1431770/78298952-aded1880-7533-11ea-85c9-a33a78378f6a.png)

**Comment by mat007** [comment]
This looks like a dup of #6212 which has more details.
I’ll go ahead and close this one, thanks for reporting!

**Comment by mat007** [comment]
Or maybe not, #6212 looks quite specific.

**Comment by mat007** [comment]
For some reason the diagnostics failed to upload, could you check the main log file `%LOCALAPPDATA%\Docker\log.txt` for errors?

**Comment by hassanselim0** [comment]
Yeah my issue isn't with Kubernetes, it's with the Settings UI in the dashboard.
I uploaded another diagnostic, here is the ID: 740ABFC4-4FD7-42DF-A3EA-FDA214A8EE32/20200403173703
For some reason I'm unable to upload the log file here.

**Comment by hassanselim0** [comment]
To be clear, the issue persists even after I disable Kubernetes from docker's settings.json file then restarting docker. I try to open the settings page and it still loads forever.

**Comment by sintetico82** [comment]
Me too here.
This is my Diagnostics ID: A1A86023-9BB8-4F4E-931B-BC489042FE7E/20200407103106

**Comment by hassanselim0** [comment]
Update: After I restarted my laptop, the settings page now works.
Last time after I disabled Kubernetes (by editing the AppData/Roaming/Docker/settings.json file), I restarted Docker Desktop and indeed Kube's status was removed from the dashboard, but the Settings page was still loading forever.
After I restarted the laptop (it was a windows update) the settings screen seems to work as expected now.
